cob_bringup/components/rplidar_mini.launch

```diff
@@ -0,0 +1,9 @@
+<launch>
+  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
```

Comment:
this is a new dependency to `rplidar_ros`. We should add this to the package.xml and CMakeLists.txt. A requirement for new dependencies is that they have to be released for indigo and kinetic. If not, we cannot add this node to cob_bringup.

Reply:
`rplidar_ros` is already present in the [package.xml](https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/package.xml) file and is available for both [indigo and kinetic](http://wiki.ros.org/rplidar).

Reply:
oh sorry, didn't see that.